doc/geany.txt

```diff
@@ -3265,17 +3199,6 @@ same as the ones which can be used with the ANSI C strftime function.
 
 All of these settings can also be changed in the print dialog just before
 actual printing is done.
-On Unix-like systems the provided print dialog offers a print preview. The
-preview file is opened with a PDF viewer and by default GTK uses ``evince``
-for print preview. If you have not installed evince or just want to use
-another PDF viewer, you can change the program to use in the file
-``.gtkrc-2.0`` (usually found in your home directory). Simply add a line
-like::
-
-    gtk-print-preview-command = "epdfview %f"
-
-at the end of the file. Of course, you can also use xpdf, kpdf or whatever
-as the print preview command.
```

Comment:
No idea how this works in GTK+3, I guess adding a new Tool to the preferences dialog like the other external Tools.